lib/posix-event/eventfd.c

```diff
@@ -292,9 +293,25 @@ static int eventfd_vfscore_poll(struct vnode *vnode, unsigned int *revents,
 	return 0;
 }
 
+static int eventfd_vfscore_ioctl(struct vnode *vnode __unused,
+				 struct vfscore_file *fp __unused,
+				 unsigned long request, void *buf __unused)
+{
+
+	switch (request) {
+	case FIONBIO:
+		/* eventfd operations look for O_NONBLOCK flag. We just
```

Comment:
I would expand this to say that `FIONBIO` is the `ioctl` flag corresponding to the `O_NONBLOCK` flag. Otherwise, it seems weird to mention `O_NONBLOCK` with no reference to `FIONBIO`.